src/app/Fake.BuildServer.TeamCity/TeamCity.fs

```diff
@@ -39,11 +39,13 @@ module TeamCityImportExtensions =
 /// The general documentation on how to use CI server integration can be found [here](/buildserver.html).
 /// This module does not provide any special APIs please use FAKE APIs and they should integrate into this CI server.
 /// If some integration is not working as expected or you have features you would like to use directly please open an issue. 
+/// 
+/// For more information on TeamCity interaction from builc scripts [see here](https://confluence.jetbrains.com/display/TCD18/Build+Script+Interaction+with+TeamCity)
```

Comment:
I'd assume more important would be how to set the "fail on `traceError` flag" and how fake in particular interacts with TeamCity. I don't think just the link to teamcity docs helps fake users here.

I'd love to see another PR to improve the docs but I'm not blocking this one